internal/pkg/service/common/servicectx/servicectx_test.go

```diff
@@ -28,25 +28,25 @@ func TestProcess_Add(t *testing.T) {
 	op3.Add(1)
 
 	// Do some work, operations run in parallel
-	proc.Add(func(ctx context.Context, errCh chan<- error) {
+	proc.Add(func(ctx context.Context, shutdown ShutdownFn) {
 		<-ctx.Done()
 		logger.Info("end1")
```

Comment:
Mame na to tieto unit testy, ktore su bez zmeny logiky, vymenil sa iba argument.